docs/source/backend.rst

```diff
@@ -3,44 +3,88 @@
 torchaudio.backend
 ==================
 
+Overview
+~~~~~~~~
+
 :mod:`torchaudio.backend` module provides implementations for audio file I/O, using different backend libraries.
 To switch backend, use :py:func:`torchaudio.set_audio_backend`. To check the current backend use :py:func:`torchaudio.get_audio_backend`.
 
-.. warning::
-   Although ``sox`` backend is default for backward compatibility reason, it has a number of issues, therefore it is highly recommended to use ``sox_io`` backend instead. Note, however, that due to the interface refinement, functions defined in ``sox`` backend and those defined in ``sox_io`` backend do not have the same signatures.
+There are currently four implementations available.
```

Comment:
I'd mention which one is the default and whether users actually need to worry about this in most cases

Reply:
Also write out why you might care about reading this section and give an overview of its content.

Reply:
Okay, so the description here was basically duplications from description of each backend. So I removed the description here and made the list into links to the backends. That way, the list here will only give the necessary information (default/deprecated).

So the `Overall` section will talk about stuff things across backends.